website/Makefile

```diff
@@ -1,35 +1,30 @@
-VERSION?="0.3.34"
+configure_cache:
```

Comment:
Note: we usually use hyphens for tasks like this. `configure-cache`.

Though given the simplicity of this task I wonder if its worth just doing a `@mkdir -p tmp/cache` directly in the build/website tasks.